packages/synapse-react-client/package.json

```diff
@@ -220,6 +220,7 @@
     "react-test-renderer": "^18.2.0",
     "rollup-plugin-polyfill-node": "^0.10.2",
     "storybook": "^7.0.23",
+    "storybook-addon-designs": "7.0.0-beta.2",
```

Comment:
This was supposed to be added in #333 but must have gotten dropped in a merge commit